src/linker/Linker/Driver.cs

```diff
@@ -633,6 +645,8 @@ protected int SetupContext (ILogger customLogger = null)
 			//   dynamically adds steps:
 			//     ResolveFromXmlStep [optional, possibly many]
 			//     BodySubstituterStep [optional, possibly many]
+			//     XmlCustomAttributesStep [optional, possibly many]
+			// XmlCustomAttributesStep [optional, possibly many]
```

Comment:
Why add it twice?

Reply:
BlackList step adds XmlCustomAttributesStep based on the embedded xml on the assemblies